thrust/iterator/output_writer_iterator.cuh

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
This file will need to be renamed. Thrust uses the `.h` extension instead of `.cuh` for all headers.